spec/order_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'order'
+
+describe 'Order' do
+  subject(:order) { Order.new('pizza') }
+
+  it 'creates new instances of Order class' do
+    expect(order).to be_an_instance_of(Order)
+  end
+
+  it 'initialises with a specific menu' do
+    expect(order.menu).to be_an_instance_of(Menu)
+  end
+
+  describe '# add' do
+    it 'adds an item to order' do
+      order.add('Anchoa')
+      expect(order.current_order.length).to eq 1
+    end
+
+    it 'should only add items from the initialised menu' do
```

Comment:
You could make this slightly more readable. You already have the #add describe block, so probably dont need the 'add' in the it block (nit-picking)